### ISSUE TITLE
Suggestions are still popping up when package is uninstalled

### ISSUE THREAD

**Issue by samueleaton** [tushortz/Ruby-Completions#2]
When the package is uninstalled a suggestion menu is still popping up. I tried restarting sublime and it still happens.

**Comment by tushortz** [comment]
I'll look into it. What  os are you using. And did you install from package control manager or extracted it from GitHub @samueleaton

**Comment by samueleaton** [comment]
Im on OSX El Capitan. I used the package installer. Thanks.

**Comment by tushortz** [comment]
@samueleaton Try typing something on and if any suggestion pops up with func at the right hand side then its my package otherwise its the default `ruby` completion. 

Also try to disable the package, if `Ruby completion` shows up among the package then it might be that this package is installed somewhere else. I works fine on my windows and Linux. I don't have a Mac so its not easy to test quickly.

If you can send me a screenshot

**Comment by samueleaton** [comment]
I actually just fixed it. I looked at my Sublime console after trying to reinstall the package and it was showing some other errors.  I apparently had my user settings JSON file formatted incorrectly (I do extra stuff there with `installed_packages`) so it was causing a chain reaction of bugs. I fixed it and everything works now. 

And fyi, I am still using the package, I was just wondering why it was still working when I "supposedly" removed it. Sorry for the false alarm.
